docs/book/basics/simple_controller.md

```diff
@@ -1,144 +1,133 @@
-{% panel style="info", title="Under Development" %}
-This book is being actively developed.
-{% endpanel %}
-
 # Simple Controller Example
 
 This chapter walks through a simple Controller implementation.
 
-This is a simple example of the Controller for the ContainerSet API shown in *Simple Resource Example*.
+This example is for the Controller for the ContainerSet API shown in *Simple Resource Example*.
 
-> $ kubebuilder create resource --group workloads --version v1beta1 --kind ContainerSet
+> $ kubebuilder create api --group workloads --version v1beta1 --kind ContainerSet
 
 > pkg/controller/containerset/controller.go
 
 ## Setup
 
 {% method %}
-
-Code generation requires the following to be defined in controller.go:
- 
-- a `ProvideController` function returning an initialized `controller.GenericController`.  This
-  will be called from the controller-manager at runtime.
-- a go struct annotated with `// +kubebuilder:controller`.  This wires the controller in the `inject` package.
-
 #### ContainerSetController
 
-ContainerSetController has 3 annotations:
+ContainerSetController has a single annotation:
 
-- `// +kubebuilder:controller` registers the controller in the generated `pkg/inject` code ensuring it will be
-  started by the controller-manager main at runtime.
-- `// +kubebuilder:rbac` creates RBAC rules in the `hack/install.yaml` file created by `kubebuilder create config`.
+- `// +kubebuilder:rbac` creates RBAC rules in the `config/rbac/group_kind.yaml` files when `make` is run.
   This will ensure the Kubernetes ServiceAccount running the controller can read / write to the Deployment API
-- `// +kubebuilder:informers` starts the informer for listening to Deployment events
 
 ContainerSetController has 2 variables:
 
-- `InjectArgs` contains the clients provided to *ProvideController*
-- `containersetrecorder` contains a recorder to publish information displayed by `kubectl describe`.
+- `client.Client` is a client for reading / writing Kubernetes APIs.
+- `scheme *runtime.Scheme` is a runtime.Scheme used by the library to set OwnerReferences.
+
+#### Add
```

Comment:
`Add a controller` or `Register a new controller` ?

Reply:
The scaffolded fn is called "Add" right now.  Do you think we should change it?

Reply:
Here, this is more of a heading for the section. So just 'Add' doesn't have much context. 

Re: scaffolded fn, don't have a strong opinion about that.

Reply:
Done